Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/Contexts/TagsContext.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Backbone.ConsumerApi.Tests.Integration.Contexts;
+
+public class TagsContext
+{
+    public Dictionary<string, string> TagHashes { get; set; } = [];
```

Comment:
Things that are not shared between different step definitions don't need to be placed in a context. You can just add this as a field to the step definitions class where it is used.

Reply:
Actually... Do we need this at all? Why don't you just access the ETag directly via the response? Is it because it has a name in the step? Because the name might as well just be a way to make the step description clearer.